transcom/mymove
# B-18175 add ppm expense updates to move history

## [B-18175](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a871869)

## Summary

Adding updates to PPM expenses to the move history log. This includes changes to amount, status, reasons, and dates (SIT only).

These are to be checked by a reviewer.


### How to test

1. Login as a closeout counselor and view a move with documents submitted for all expense types (Contracted, oil, packing, rental, storage, tolls, weighing, other)
2. Review documents, and make edits to each document related to an expense type
3. Confirm the changes appear in the move history
